build/common.project.props

```diff
@@ -187,8 +187,8 @@
 
   <PropertyGroup Condition=" '$(IsNetCoreProject)' == 'true' ">
     <!-- Assembly attributes for net core projects -->
-    <!--AssemblyVersion should not contain the build number as all NuGet.Core projects had the build number as 0 every time -->
-    <AssemblyVersion>$(SemanticVersion).0</AssemblyVersion>
+    <!--AssemblyVersion should have the build number incremented for every milestone. Currently this is 2 to align with 15.3 preview2 -->
+    <AssemblyVersion>$(SemanticVersion).2</AssemblyVersion>
```

Comment:
I would also link that to this issue/explanation as to why we have it like that. 

That way we don't run into a problem when someone tries to "fix" it again at some point :)